spec/hotel_spec.rb

```diff
@@ -0,0 +1,154 @@
+require_relative 'spec_helper'
+require 'pry'
+
+describe 'Module Hotel' do
+  before do
+    @hotel = Hotel::Hotel.new
+    @start_date = Date.parse("1/9/2018")
```

Comment:
I would like to see better organization for these tests. My typical strategy is to have a big `describe` for the class, and then a smaller `describe` nested inside for each method.